plugins/kubernetesapi-report/README.md

```diff
@@ -40,6 +40,8 @@ sauron.plugins:
         "THE_OUTPUT_KEY": "the.prop.key.in.the.file"
       "[/path/to/file_b.env]":
         "ANOTHER_OUTPUT_KEY": "the.prop.key.in.the.file"
+    containersCheck:
+      - productionReadiness
```

Comment:
I don't think this approach is correct. Looking at other configurations for the same plugin, we specify exactly what we're checking for. Based on this pattern, I suggest we add the specific checks that need to be applied, rather than relying on the ambiguous meaning of the group purpose like 'productionReadiness'